ui/src/routes/_layout/organizations/$orgId.products.$productId.repositories.$repoId.create-run.tsx

```diff
@@ -987,7 +1017,83 @@ const CreateRunPage = () => {
                 <AccordionItem value='evaluator' className='flex-1'>
                   <AccordionTrigger>Evaluator</AccordionTrigger>
                   <AccordionContent>
-                    No job configurations yet implemented for Evaluator.
+                    <div className='text-sm text-gray-500'>
+                      In case any input field is left empty, the default path
+                      from the config file provider will be used for the
+                      corresponding file.
+                    </div>
+                    <FormField
+                      control={form.control}
+                      name='jobConfigs.evaluator.ruleSet'
+                      render={({ field }) => (
+                        <FormItem className='pt-4'>
+                          <FormLabel>Evaluator rules</FormLabel>
+                          <FormControl>
+                            <Input {...field} />
+                          </FormControl>
+                          <FormDescription>
+                            The path to the rules file which is resolved from
+                            the configured configuration source. Note: in case
```

Comment:
Start capitalized after ":", so "Note: In ...".

Reply:
Are we Americans now?! :D

Reply:
My thoughts exactly.

Reply:
We use American, not British, English in all text. So yes 😉